### ISSUE TITLE
Support for Visual Studio

### ISSUE THREAD

**Issue by dbnex14** [pleriche/FastMM4#52]
Any chance to at support for Visual Studio?

**Comment by baka0815** [comment]
You know that this is a memory manager specially for Delphi/Pascal/FPC?

**Comment by dbnex14** [comment]
Yes I do but I was so impressed with it while I was working in Delphi that I find out would be great to have it posted to. Net.
While . Net has few tools, I find none of them to be as practical as FastMM.
It would be fantastic to have such a great tool in .Net.
